src/TourManager/Api/Controllers/TourController.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace Travely.TourManager.Api.Controllers
+{
+    [Route("api/Tour")]
```

Comment:
Use `[controller]` instead of hardcoding

Reply:
ok